packages/cli/src/util/get-user.ts

```diff
@@ -2,11 +2,7 @@ import Client from './client';
 import { User } from '../types';
 import { APIError, InvalidToken, MissingUser } from './errors-ts';
 
-let user: User | undefined;
-
 export default async function getUser(client: Client) {
-  if (user) return user;
```

Comment:
This used to be cached?

Reply:
It did, but for not really any reason. This function is invoked exactly once for most commands (and if it's > 1 then the command should be updated to not do that). Also the caching was getting in the way of the unit tests getting unexpectedly cached results 😛